src/main/java/com/google/sps/servlets/NewsServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.Article;
+
+/**
+ * Servlet for generating news articles from keywords.
+ */
+@WebServlet("/news")
+public class NewsServlet extends HttpServlet {
+  private List<String> keywords = new ArrayList<>();
+  
+  // Hardcoded list of articles. 
+  // In future implementation, will be generated based on key words.
+  private List<Article> articles = new ArrayList<>(
+      List.of(
+        new Article.Builder("Bolton says Trump turned a blind eye to the coronavirus pandemic",
```

Comment:
[Nit] Multiline function calls should be indented by 4 spaces.  Chain function are indented by 2 spaces.  Therefore, it should look something like this (to be honest I'm not a huge fan of this formatting but it is what is recommended by the Googel Java style guide).

```
myFunction(
    parameter1,
    parameter2,
    parameter3)
  .chainedFunction(parameter)
  .chainedFunction2(parameter)
```